base/src/main/java/com/smartdevicelink/managers/screen/choiceset/BaseChoiceSetManager.java

```diff
@@ -646,9 +698,31 @@ boolean setUpChoiceSet(ChoiceSet choiceSet) {
     }
 
     KeyboardProperties defaultKeyboardConfiguration() {
```

Comment:
Please revert all changes to this method as they are not necessary with the new `createValidKeyboardConfigurationBasedOnKeyboardCapabilitiesFromConfiguration` that is suggested in another comment.